components/input/gpio/gpio_test.go

```diff
@@ -386,7 +481,7 @@ func TestGPIOInput(t *testing.T) {
 			test.That(tb, atomic.LoadInt64(&s.axis1Callbacks), test.ShouldEqual, 0)
 		})
 
-		s.b.Analogs["analog1"].Set(1023)
+		s.analog1.Write(s.ctx, 1023, nil)
```

Comment:
obviously this is passing lint checks, so no action needed, but I'm surprised it's fine to not check the error returned here... maybe because it's an inject?